platform/core/src/utils/subscribeToNextViewportGridChange.ts

```diff
@@ -24,10 +24,6 @@ function subscribeToNextViewportGridChange(
         viewportGridService.EVENTS.LAYOUT_CHANGED,
         callback
       ),
-      viewportGridService.subscribe(
-        viewportGridService.EVENTS.GRID_STATE_CHANGED,
```

Comment:
At the very least the doc comment for this utility function needs to be updated to exclude the `GRID_STATE_CHANGED` event.